pubspec.yaml

```diff
@@ -10,14 +10,14 @@ environment:
 dependencies:
   flutter:
     sdk: flutter
-  intl: ^0.17.0
+  intl: ^0.18.0
   plugin_platform_interface: ^2.0.2
   universal_io: ^2.0.4
 
 dev_dependencies:
   flutter_test:
     sdk: flutter
-  flutter_lints: ^1.0.4
+  flutter_lints: ^2.0.1
```

Comment:
Can we downgrade this version? Additionally, we want to continue supporting Flutter 2.10.5

Reply:
Ive downgraded the flutter_lints dependency as requested. 
For the new flutter version, the minimal version is 2.12.0. 
<img width="596" alt="CleanShot 2023-05-18 at 23 03 14@2x" src="https://github.com/gokberkbar/locale_plus/assets/16548374/6d4d9fa0-4206-4ed5-83e8-28f2cd4bc103">